### PR TITLE
[console-wallet] Add whois command

### PR DESCRIPTION
**Pull request by delta1**
## Description

Adds a `whois` command to the console wallet command/script runner,
which takes a public key or emoji id, and then displays both. Useful for
getting a public key from emoji id or vice versa.

## Motivation and Context

Needed an easy way of converting pubkey <-> emoji id 

## How Has This Been Tested?

`tari_console_wallet --command "whois 🌠🔧🎧🐗🌟📡💊👛🌙🎉🎸👒🌙😿🐚😹💻😍🔩🐲🎻🚗🍰🎵🐀💻🐍👠😿🐚🔫🎃📡"`

output: 

```
==============
Command Runner
==============
1. whois 0ad74c7509ccaf9f0643599606eb78e9c4e5d9875cf433576bc46fa2eb78db40
Public Key: 0ad74c7509ccaf9f0643599606eb78e9c4e5d9875cf433576bc46fa2eb78db40
Emoji ID  : 🌠🔧🎧🐗🌟📡💊👛🌙🎉🎸👒🌙😿🐚😹💻😍🔩🐲🎻🚗🍰🎵🐀💻🐍👠😿🐚🔫🎃📡
```

## Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [x] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [ ] Documentation

## Checklist:

* [x] I'm merging against the `development` branch.
* [x] I ran `cargo-fmt --all` before pushing.
* [ ] I ran `cargo test` successfully before submitting my PR.
* [x] I have squashed my commits into a single commit.
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

File: applications/tari_console_wallet/src/automation/command_parser.rs [hunk 1]
```diff
@@ -97,11 +98,25 @@ pub fn parse_command(command: &str) -> Result<ParsedCommand, ParseError> {
...
+    let mut parsed_args = Vec::new();
+
+    // public key/emoji id
+    let pubkey = args
+        .next()
+        .ok_or_else(|| ParseError::Empty("public key or emoji id".to_string()))?;
+    let pubkey = parse_emoji_id_or_public_key(pubkey).ok_or(ParseError::PublicKey)?;
+    parsed_args.push(ParsedArgument::PublicKey(pubkey));
+
+    Ok(parsed_args)
```
Comment (sdbondi) [inline applications/tari_console_wallet/src/automation/command_parser.rs:117]: (ignore this) nit:

```suggestion
 let pk_arg = args
        .next()
        .ok_or_else(|| ParseError::Empty("public key or emoji id".to_string()))
        .and_then(|pk| parse_emoji_id_or_public_key(pk).ok_or(ParseError::PublicKey))
        .map(ParsedArgument::PublicKey)?;

    Ok(vec![pk_arg])
```

**Review by sdbondi — APPROVED** [review]
Tested - works great!

**Review by philipr-za — APPROVED** [review]
LGTM

File: applications/tari_console_wallet/src/automation/command_parser.rs (hunk 1 above)
Comment (delta1) [reply applications/tari_console_wallet/src/automation/command_parser.rs:117]: That's awesome. Will refactor some time.

**Pull request merged**